manifest.toml

```diff
@@ -0,0 +1,98 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "diaspora"
+name = "Diaspora"
+description.en = "Distributed social networking service"
+description.fr = "Service de réseau social distribué"
+
+version = "0.7.18.2~ynh2"
+
+maintainers = ["rafi59"]
+
+[upstream]
+license = "AGPL-3.0"
+website = "https://diasporafoundation.org/"
+admindoc = "https://wiki.diasporafoundation.org/FAQ_for_pod_maintainers"
+userdoc = "https://wiki.diasporafoundation.org/FAQ_for_users"
+code = "https://github.com/diaspora/diaspora"
+
+[integration]
+yunohost = ">=11.2"
+architectures = "all" # FIXME: can be replaced by a list of supported archs using the dpkg --print-architecture nomenclature (amd64/i386/armhf/arm64), for example: ["amd64", "i386"]
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M" # FIXME: replace with an **estimate** minimum disk requirement. e.g. 20M, 400M, 1G, ...
```

Comment:
419M for my actual `/var/www/diaspora` without uploads, so maybe a bit more for ruby and other deps if we count them? 500M I'd say.

Reply:
Yeah, indeed. Fixed in master directly.